plugins/generator-1.19.4/forge-1.19.4/templates/json/block_loot_table.json.ftl

```diff
@@ -0,0 +1,66 @@
+<#include "../mcitems.ftl">
+<#assign defaultSlabDrops = data.blockBase?has_content && data.blockBase == "Slab" && !(data.customDrop?? && !data.customDrop.isEmpty())/>

```

Comment:
Maybe these templates can be unified? We just need a couple more checks (`data.blockBase??` where blocks and plants are distinguished - here and double block check) for a cost of 3 templates less.

Reply:
That's probably doable